nengo_loihi/config.py

```diff
@@ -11,6 +11,12 @@ def add_params(network):
       * ``on_chip``: Whether the ensemble should be simulated
         on a Loihi chip. Marking specific ensembles for simulation
         off of a Loihi chip can help with debugging.
+    `nengo.Connection`
+      * ``pop_type``: The axon format when using population spikes, which are only
+        used for convolutional connections. By default, we use ``pop_type`` 32.
+        Setting ``pop_type`` to 16 allows more axons to fit on one chip as long as
+        the ``Convolution`` transform has ``channels_last=True`` and ``n_filters``
+        is a multiple of 4.
```

Comment:
We can actually use `pop_type=16` for any type of convolutional connection, it just requires making up to 3 additional copies of the weights to deal with the different offsets. I implement this in dce42a6e.

Reply:
I just copied this info from the former changelog entry that seemed to indicate that the space savings would come into effect in this situation. Is that not the case? If it is always advantageous to use `pop_type=16` then why don't we just do that and remove all the complexity added to support both types?

Reply:
Looking at the tests in `test_conv`, it looks like a lot of them use `n_filters` as a multiple of 4, a holdover from when we did have this restriction. We should probably change some of them to use different values (or test a few values). I'm looking particularly at `test_chip_population_axons` and `test_conv_deepnet`.

Reply:
That makes sense. Can you answer my question about whether `pop_type=16` is always better to use or are there advantages to `pop_type=32`?

Reply:
Looking at dce42a6 again, I think it just implements the copying for cxBase > 256, not for pop16 when `cxBase % 4 != 0`. There's no reason we can't implement the same copying trick to get pop16 to work everywhere, but I don't think I've actually done that anywhere yet.

The reason we don't want to use pop16 always is that the weight copying requires more weight memory. So it's a tradeoff: fewer axons but more weight memory with pop16, or more axons but less memory with pop32. Or you can set up your network in a way that will work with pop16 without any copying (namely `channels_last=True` and `n_filters % 4 == 0` for all layers).

Reply:
I just removed the note about space savings as it seemed too complicated. Anyone wanting to change the `pop_type` will presumably be doing it for their own specific reason.